cli/internal/cmd/operators_list.go

```diff
@@ -53,22 +54,22 @@ func listOperators(_ *cobra.Command, _ []string) {
 	}
 
 	// verify the signatures of all the operators
-	var addresses []string
+	var operatorIdentities []crypto.Identity
```

Comment:
Given you know the size of the `operators` array, this is probably the kind of cases where you either want to:
- use a fixed size array, but no one does that usually
- use a slice made with `make([]crypto.Identity,0, len(operators))` so that it's capacity is already correct, to avoid `append` having to allocate a bigger array in each iteration